src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -185,13 +189,13 @@ const JobHistory = _.flow(
 
   const sortedSubmissions = _.orderBy(
     Utils.cond(
-      [sort.field === 'entityName', ['submissionEntity.entityName']],
-      [sort.field === 'numberOfWorkflows', [s => _.sum(_.values(s.workflowStatuses))]],
-      [sort.field === 'status', [s => {
+      [sort.field === 'entityName', () => 'submissionEntity.entityName'],
```

Comment:
@breilly2 This code was throwing deprecation warnings.

Reply:
Cool. Also, this could actually be `Utils.switchCase`, as Isaac pointed out to me in a similar situation in the showcase PR.

Reply:
@breilly2 @zarsky-broad I decided to stop being lazy and changed this to Utils.switchCase.